packages/expo-image/android/build.gradle

```diff
@@ -0,0 +1,76 @@
+// android/build.gradle
+
+// based on:
+//
+// * https://github.com/facebook/react-native/blob/0.60-stable/template/android/build.gradle
+//   original location:
+//   - https://github.com/facebook/react-native/blob/0.58-stable/local-cli/templates/HelloWorld/android/build.gradle
+//
+// * https://github.com/facebook/react-native/blob/0.60-stable/template/android/app/build.gradle
+//   original location:
+//   - https://github.com/facebook/react-native/blob/0.58-stable/local-cli/templates/HelloWorld/android/app/build.gradle
+
+def DEFAULT_COMPILE_SDK_VERSION = 28
+def DEFAULT_BUILD_TOOLS_VERSION = '28.0.3'
+def DEFAULT_MIN_SDK_VERSION = 16
+def DEFAULT_TARGET_SDK_VERSION = 28
+
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven'
```

Comment:
Why aren't you using `kotlin` language? We agreed on migrating towards `kotlin`, aren't we? 🤔

Reply:
I don't think it has a status of a requirement, more like a recommendation.

Using Kotlin in this case will make the development slower (I don't consider myself fluent in that language) and could make the users' apps bigger (+kotlin bindings).

If we decide to do so, after the first stage of rapid development we may reconsider migrating this module to Kotlin.

Reply:
As for now I cannot support my suggestion by any requirement, so let it be as you say, but personally I'm against for using `java` 😞